Gemfile

```diff
@@ -41,7 +41,7 @@ gem "fog-aws", "~> 3.12" # 'fog' gem to support Amazon Web Services
 gem "front_matter_parser", "~> 1.0" # Parse a front matter from syntactically correct strings or files
 gem "gemoji", "~> 4.0.0.rc3" # Character information and metadata for standard and custom emoji
 gem "gibbon", "~> 3.4" # API wrapper for MailChimp's API
-gem "hairtrigger", "~> 0.2.24" # HairTrigger lets you create and manage database triggers in a concise, db-agnostic, Rails-y way.
+gem "hairtrigger", github: "citizen428/hair_trigger", branch: "master" # HairTrigger lets you create and manage database triggers in a concise, db-agnostic, Rails-y way.
```

Comment:
We're temporarily using my fork until upstream merges the PR with Rails 7 support.

Reply:
merged - in 0.2.25  - see https://github.com/forem/forem/pull/16317

Reply:
@djuber I actually had subscribed to the PR over a the hairtrigger repo, but thanks!